cli/cmd/update_project.go

```diff
@@ -47,29 +46,29 @@ For more information about updating projects or upstream repositories, please go
 			return errors.New("required argument PROJECTNAME not set")
 		}
 
-		if !keptncommon.ValidateKeptnEntityName(args[0]) {
-			errorMsg := "Project name contains upper case letter(s) or special character(s).\n"
-			errorMsg += "Keptn relies on the following conventions: "
-			errorMsg += "start with a lower case letter, then lower case letters, numbers, and hyphens are allowed.\n"
-			errorMsg += "Please update project name and try again."
-			return errors.New(errorMsg)
-		}
 		return nil
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
+
+		shipyard, err := retrieveShipyard(*updateProjectParams.Shipyard)
```

Comment:
Can this code be extracted and shared with the create project function?

Reply:
can be, sure